setup.cfg

```diff
@@ -60,6 +60,7 @@ install_requires =
     optuna>=2.0.0
     pandas>=1.0.0
     tabulate
+    pystow
```

Comment:
additional dependency?

Reply:
See explanation in https://github.com/pykeen/pykeen/pull/192

Reply:
Okay you were right to be skeptical this made no sense in this PR